tests/unit/jira/test_projects.py

```diff
@@ -439,72 +510,33 @@ def test_get_project_issues_with_search_mixin(projects_mixin):
     projects_mixin.jira.jql.assert_not_called()
 
 
-def test_get_project_issues_without_search_mixin(projects_mixin):
```

Comment:
Removing `test_get_project_issues_without_search_mixin` is appropriate as discussed in #300.